### DISCUSSION TITLE
About the xml cars

### DISCUSSION THREAD

**Discussion by Drsexo** [YimMenu/YimMenu#1642] [General]
Where do we put the xml files for them to be detected ? #1594

**Comment by Drsexo** [comment]
nvm found it if anyone is wondering the same thing its : AppData\Roaming\YimMenu\xml_vehicles

**Comment by Maxi23-1** [comment]
what cars do you have on xml

**Reply by xesdoog** [reply]
Here's my list
[xml_vehicles.zip](https://github.com/YimMenu/YimMenu/files/14516623/xml_vehicles.zip)

**Reply by heni015** [reply]
From different sources, only few maybe didn't work, but most working of 2488 vehicles.
Compared by text length and deleted doubles with different filenames. :D Have fun
[xml_vehicles.zip](https://github.com/YimMenu/YimMenu/files/15195371/xml_vehicles.zip)
